ALICE3/Tasks/pidRICHqa.cxx

```diff
@@ -205,6 +216,23 @@ struct richPidQaMc {
     }
   }
 
+  template <uint8_t pidIndex, typename T, typename TTT, typename TT>
+  void fillNsigmafRICH(const T& track, const TTT& particle, const TT& mcParticles, const float& nsigma)
```

Comment:
I don't see `mcParticles` used anywhere.

Reply:
Indeed it was used in the isphysical primary

Reply:
This I fixed